Kconfig.nrf

```diff
@@ -8,6 +8,12 @@ rsource "subsys/net/openthread/Kconfig.defconfig"
 
 menu "Nordic nRF Connect"
 
+# Override configuration from zephyr which sets this to 0x200 if MCUboot is
+# enabled (CONFIG_BOOTLOADER_MCUBOOT), since NCS use partition_manager to
+# get this offset intsead.
+config ROM_START_OFFSET
+	default 0
```

Comment:
could we safeguard this ?

Samples may be built as single image, in which case partition manager is not activated (unless explicitly enabled).
We have started sysbuild integration and will slowly transition samples, so this default may not be appropriate in all cases.

Reply:
Forcing this to 0 would actually entirely break sysbuild in ncs.

Reply:
Added check for `PARTITION_MANAGER_ENABLED`

Reply:
This should be 0 for single image builds, as it is used when you need an empty space to fill with something else, so the use cases would be if the nrf5 bootloader Kconfig is set then this would be non-zero, or if you are building a application for mcuboot without mcuboot (which, in ncs, is not possible as it will invoke a child image for mcuboot if the Kconfig is enabled)